doc/api/util.md

```diff
@@ -270,6 +270,26 @@ util.formatWithOptions({ colors: true }, 'See object %O', { foo: 42 });
 // when printed to a terminal.
 ```
 
+## util.getNodeReport()
```

Comment:
This accepts an optional `Error` argument, correct?

Reply:
addressed, the whole section is modified and now includes this optional argument scenario as well.